cassandra/README.md

```diff
@@ -41,45 +42,37 @@ We can now exit the shell by simply exiting.
 ```
 exit
 ```
-At this stage in the backup and restore, we want to ensure that the data we just created will be consistent and reliable.
-In order to achieve that, we need to able to quiesce cassandra. We can achieve this in cassandra by using a cassandra tool called
-nodetool. This will allow us to flush memory onto what are called sstables that store the data in immutable files. 
-Nodetool also has features that verifies the files were flushed and replicated across the nodes.
-
 To start the quiesce feature in cassandra, we need to run the command Drain that flushes all memory from memtables in cassandra into
-immutable sstables. Drain will also stop listening for connections from the client and other nodes.
-
-Verify first that the data isn't in our sstabless yet. There should be no output printed after running nodetool getsstables.
-Once that happens, then drain the data to them.
+immutable sstables. Note this is specific to how cassandra handles storing data. Drain will also stop listening for connections from the client and other nodes.
+nodetool. Nodetool will also be used to verifiy the data were flushed to files and replicated across the nodes at the end.
 
+From here we could manually quiesce cassandra by doing (`oc exec -it cassandra-0 -- nodetool drain` but when we do our backup, we have used velero hooks to do that instead pre-backup.
+## Back up the application.
 ```
-oc exec -it cassandra-0 -- nodetool getsstables classicmodels offices 1
-oc exec -it cassandra-0 -- nodetool drain
+ansible-playbook backup.yaml
 ```
-Now before we take a backup of cassandra, lets also verify that cassandra has stop
-listening for connections. Try to access the cqlsh again. The following command should be an 
+After taking the backup of cassandra, lets also verify that cassandra has stop
+listening for connections. This means writes have also stopped occurring temporarily. Try to access the cqlsh again. The following command should be an 
 error like the following below after executing the command.
```

Comment:
So how do we bring Cassandra back into an operational state after the drain command? I think that we also need a post hook which will bring the cluster up is that correct?

Reply:
Changed to flush that will disable writes during backup and enable writes again after backup.